code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -68,6 +67,76 @@
 			return -1
 	return 0
 
+
+
+//Organ
+
+/obj/item/organ/heart/nightmare
+	name = "heart of darkness"
+	desc = "An alien organ that twists and writhes when exposed to light."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "demon_heart-on"
+	color = "#1C1C1C"
+	var/respawn_progress = 0
+	var/obj/item/light_eater/blade
+
+
+
+/obj/item/organ/heart/nightmare/attack(mob/M, mob/living/carbon/user, obj/target)
+	if(M != user)
+		return ..()
+	user.visible_message("<span class='warning'>[user] raises [src] to their mouth and tears into it with their teeth!</span>", \

```

Comment:
Replace "their" with `[user.p_their()]`?

Reply:
^ this

Reply:
I am pretty sure Kor has decided against doing this, as he has added more of these messages.